tests/testapp/urls.py

```diff
@@ -8,3 +8,13 @@
     path("_pictures/", include("pictures.urls")),
     path("admin/", admin.site.urls),
 ]
+
+# django-rq
+try:
+    import django_rq  # NoQA
+except ImportError:  # django_rq is not installed
+    pass
+else:
+    urlpatterns += [
+        path("django-rq/", include("django_rq.urls")),
+    ]
```

Comment:
Not sure if this is really needed, since we don't have to access the URLs in our test setup, right?